pkg/api/validation/validation.go

```diff
@@ -753,6 +761,88 @@ func validateDownwardAPIVolumeSource(downwardAPIVolume *api.DownwardAPIVolumeSou
 	return allErrs
 }
 
+func validateProjectionsVolumeSource(projection *api.Projections, fldPath *field.Path) field.ErrorList {
```

Comment:
I tried to be consistent with the naming. Just to be clear, you're talking about changing Projections to Projection throughout? https://github.com/kubernetes/community/pull/169/commits/1383c030484ee693df59cf8b40255954016b2f27#diff-c5fe36fe223ccc59602a515f60c5db37R189